### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#239]
Onboarding screens

1. Welcome to StatsCAN should be read as heading
2. Terms of Use should be read as heading and get the initial focus when coming from welcome screen
3. The whole description of the "Terms of Use" should be read in paragraphs instead of reading as a one big paragraph.
4. When Swiping left to right on "Terms of Use" screen, the focus should not go to the scroll bar and it should continue reading one paragraph after another.
5.  Accept should be labelled as "Accept Terms of Use" button.
6. When the user accepts the Terms of Use, the focus on the next screen should go to Publications heading.

Publications screen
1. Notifications should be labelled as "Notifications" only as it says button by itself on iOS.
2. Latest should be read as a heading 
3. When reading a card it should first read the title of the card and then add a comma "," for a pause and then read the subject.

Notifications screen (No notifications)
1. When swiping from left to right, the focus goes on to the three dots even though they are not visible. (should be removed)

Level - 3
1. Back arrow at the bottom should labelled as "back" button
2. Forward arrow at the bottom should be labelled as "Forward" button

Search screen (First time)
1. On the search screen, when swiping from left to right, the focus should not go to the image inside the search field. This applies to Recent searches, when the user is searching and when there are no results to be displayed.
2. There should be no Focus on the decorative image.
3. Cross icon should be labelled as "clear text icon" button as it is saying button two times.
4. When searching, clicked on filter by subjects, the Focus should go to "Filter by subject(s)" heading > Close button
5. When the Focus goes to a particular subject, it should be say the subject name and the state of the checkbox.

For you screen (Intro)
1. The focus should go to the "For you" heading and then go to More options. (It can be tested by activating voiceover on For you page).
2. On the Following screen, select subjects should be labelled as button and it should highlight the whole button instead if Select subjects.
3. On the Saved screen, Go to publications should be labelled as a button as well and should be highlighted as a whole.

Settings screen
1. When a language is selected it should read as "name Language selected " button. For instance if the English language is selected it should read as "English language selected "
2. The other unselected option should be read as "French language" button and vice-versa.
2. Colour theme - Same behaviour as of Language.

Settings > Manage subjects 
1. Manage subjects heading should get the first focus and the "Go back" button.
2. When the focus goes to a subject, it should read the subject name and the state of the checkbox "checked/unchecked".

Settings > Contact us
1. The heading should always get the first focus and then "Go back" button.
2. When swiped from left to right, the focus goes to hidden link "Skip to main content", it should be disabled.
3. When reading a link, it should read the text first and then point it out as a link. For instance, "Privacy notice, link".
4. And it should highlight the whole paragraph.

Settings > Terms of Use
1. The whole description of the "Terms of Use" should be read in paragraphs instead of reading as a one big paragraph.

**Comment by zhuelle** [comment]
@payeisa need prioritization.

**Comment by payeisa** [comment]
@zhuelle Would it be possible to get a t-shirt size for this one?

**Comment by zhuelle** [comment]
@Rajat-Sen It looks like there are many accessibility issues stated in this ticket. Please verify if they are still all valid at the current stage. If they are valid, please break it down into multiple tickets so that one ticket contains one issue. Once this is down the dev team can estimate the effort for each problem. This ticket can be closed once the break-down is completed and links all break-down tickets to this ticket. Thanks!

**Comment by payeisa** [comment]
Priority added.

**Comment by zhuelle** [comment]
@Rajat-Sen Do we still need this ticket given that you have divided this ticket into sub-tickets? If not, please close it. Thanks!

**Comment by Rajat-Sen** [comment]
@zhuelle, I closed this ticket 4 days ago!
